test/QuadForm/Torsion.jl

```diff
@@ -47,22 +47,44 @@
   L = Zlattice(matrix(ZZ, [[2,0,0],[0,2,0],[0,0,2]]))
   T = Hecke.discriminant_group(L)
   S, _ = sub(T, gens(T))
-  @test basis_matrix(Hecke.cover(Hecke.orthogonal_submodule_to(T, S)[1])) == matrix(QQ, 3, 3, [1//2,0,0,0,1//2,0,0,0,1//2])
+  @test basis_matrix(Hecke.cover(Hecke.orthogonal_submodule_to(T, S)[1])) == basis_matrix(L)
+  L1 = Zlattice(ZZ[1 0 0 0 0 0 0 0 0 0;
```

Comment:
You can replace this with `identity_matrix(ZZ,10)`

Reply:
oh ok. I didn't know how to call the identity matrix. Will do that. Thank you:)